docs/how-to/how-to-enable-storage-in-confidential-pods.md

```diff
@@ -287,48 +277,32 @@ the `cc-azurefile-csi` PVC in `/mnt/persistent-ok`.
   apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
-    name: pvc-cc-azurefile-csi
+    name: azurefile-pvc
   spec:
     accessModes:
-      - ReadWriteOnce
-    resources:
-      requests:
-        storage: 10Gi
-    storageClassName: cc-azurefile-csi
-  ---
-  apiVersion: v1
-  kind: PersistentVolumeClaim
-  metadata:
-    name: pvc-azurefile-csi
-  spec:
-    accessModes:
-      - ReadWriteOnce
+      - ReadWriteMany
     resources:
       requests:
         storage: 10Gi
     storageClassName: azurefile-csi
   ---
-  kind: Pod
   apiVersion: v1
+  kind: Pod
   metadata:
-    name: demo-cc-azurefile
+    name: kata-cc-writer
   spec:
     runtimeClassName: kata-cc-isolation
     containers:
-      - image: busybox:latest
-        name: busybox
-        volumeMounts:
-          - name: cc-azurefile-csi-vol
-            mountPath: /mnt/persistent-ok
-          - name: azurefile-csi-vol
-            mountPath: /mnt/persistent-broken
+    - name: busybox
+      image: busybox
+      command: ["sh", "-c", "echo 'Hello from kata-cc!' > /mnt/azurefile/hello-kata-cc.txt && sleep 3600"]
```

Comment:
I would remove this - let's keep the behavior of the example consistent with the others.